includes/wikia/services/tests/MediaQueryServiceTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * The following test class has been created while fixing SUS-198.
+ */
+class MediaQueryServiceTest extends WikiaBaseTest {
+
+	/**
+         * @expectedException InvalidArgumentException
+         * @expectedExceptionMessage $sort was none of 'recent', 'popular', 'trend'.
+	 */
+	public function testGetVideoListSortInvalid() {
+		$service = new MediaQueryService();
+		$list = $service->getVideoList(
+			'all',			// filter
+			1,			// limit
+			1, 			// page
+			['testProvider'],	// providers
+			['testCategory'],	// categories
+			'invalidSort'		// sort
+		);
+	}
+
+	/**
+	 * A simple sanity test.
+	 */
+	public function testGetVideoListGeneral() {
```

Comment:
Yeah. This is just a simple sanity test before I started working on that. I'd leave it as it is rather than remove it.